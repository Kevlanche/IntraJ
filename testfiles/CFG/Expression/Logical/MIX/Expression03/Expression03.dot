digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"; "MethodDecl[50]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[26]"; "Opt[27]"; "Exit[28]"; "Entry[51]"; "Modifiers[52]"; "PrimitiveTypeAccess[54]"; "List[55]"; "List[64]"; "Opt[65]"; "Exit[66]"}
{ rank = same; "List[15]"; "ParameterDeclaration[18]"; "ParameterDeclaration[22]"; "Block[28]"; "List[53]"; "ParameterDeclaration[56]"; "ParameterDeclaration[60]"; "Block[66]"}
{ rank = same; "Modifiers[19]"; "PrimitiveTypeAccess[21]"; "Modifiers[23]"; "PrimitiveTypeAccess[25]"; "List[29]"; "Modifiers[57]"; "PrimitiveTypeAccess[59]"; "Modifiers[61]"; "PrimitiveTypeAccess[63]"; "List[67]"}
{ rank = same; "List[20]"; "List[24]"; "ExprStmt[30]"; "List[58]"; "List[62]"}
{ rank = same; "MethodAccess[31]"}
{ rank = same; "List[33]"; "List[48]"; "List[49]"; "UncheckedExceptions[32]"}
{ rank = same; "AndLogicalExpr[34]"; "OrLogicalExpr[41]"}
{ rank = same; "GTExpr[35]"; "LTExpr[38]"; "GTExpr[42]"; "LTExpr[45]"}
{ rank = same; "VarAccess[36]"; "IntegerLiteral[37]"; "VarAccess[39]"; "IntegerLiteral[40]"; "VarAccess[43]"; "IntegerLiteral[44]"; "VarAccess[46]"; "IntegerLiteral[47]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Expression03 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[18]" [label="ParameterDeclaration\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[19]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[20]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[21]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[22]" [label="ParameterDeclaration\n < c >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[23]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[25]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[27]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[28]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[29]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[30]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[31]" [label="MethodAccess\n < f >", fillcolor=white   style=filled  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[34]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[35]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[36]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[37]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[38]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[39]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[40]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"OrLogicalExpr[41]" [label="OrLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[42]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[43]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[44]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[45]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[46]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[47]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"List[48]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[49]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[32]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[28]" [label="Exit\n ", fillcolor=white   style=filled  ]
"MethodDecl[50]" [label="MethodDecl\n < f >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[51]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[52]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[53]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[54]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[55]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[56]" [label="ParameterDeclaration\n < x >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[57]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[58]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[59]" [label="PrimitiveTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[60]" [label="ParameterDeclaration\n < y >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[61]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[62]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[63]" [label="PrimitiveTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[64]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[65]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[66]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[67]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[66]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"Modifiers[19]" -> "List[20]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "Modifiers[19]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "PrimitiveTypeAccess[21]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[18]" [style=dashed, color=gray]
"Modifiers[23]" -> "List[24]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "Modifiers[23]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "PrimitiveTypeAccess[25]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[22]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[26]" [style=dashed, color=gray]
"GTExpr[35]" -> "VarAccess[36]" [style=dashed, color=gray]
"GTExpr[35]" -> "IntegerLiteral[37]" [style=dashed, color=gray]
"AndLogicalExpr[34]" -> "GTExpr[35]" [style=dashed, color=gray]
"LTExpr[38]" -> "VarAccess[39]" [style=dashed, color=gray]
"LTExpr[38]" -> "IntegerLiteral[40]" [style=dashed, color=gray]
"AndLogicalExpr[34]" -> "LTExpr[38]" [style=dashed, color=gray]
"List[33]" -> "AndLogicalExpr[34]" [style=dashed, color=gray]
"GTExpr[42]" -> "VarAccess[43]" [style=dashed, color=gray]
"GTExpr[42]" -> "IntegerLiteral[44]" [style=dashed, color=gray]
"OrLogicalExpr[41]" -> "GTExpr[42]" [style=dashed, color=gray]
"LTExpr[45]" -> "VarAccess[46]" [style=dashed, color=gray]
"LTExpr[45]" -> "IntegerLiteral[47]" [style=dashed, color=gray]
"OrLogicalExpr[41]" -> "LTExpr[45]" [style=dashed, color=gray]
"List[33]" -> "OrLogicalExpr[41]" [style=dashed, color=gray]
"MethodAccess[31]" -> "List[33]" [style=dashed, color=gray]
"MethodAccess[31]" -> "List[48]" [style=dashed, color=gray]
"MethodAccess[31]" -> "List[49]" [style=dashed, color=gray]
"MethodAccess[31]" -> "UncheckedExceptions[32]" [style=dashed, color=gray]
"ExprStmt[30]" -> "MethodAccess[31]" [style=dashed, color=gray]
"List[29]" -> "ExprStmt[30]" [style=dashed, color=gray]
"Block[28]" -> "List[29]" [style=dashed, color=gray]
"Opt[27]" -> "Block[28]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[27]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[28]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"MethodDecl[50]" -> "Entry[51]" [style=dashed, color=gray]
"Modifiers[52]" -> "List[53]" [style=dashed, color=gray]
"MethodDecl[50]" -> "Modifiers[52]" [style=dashed, color=gray]
"MethodDecl[50]" -> "PrimitiveTypeAccess[54]" [style=dashed, color=gray]
"Modifiers[57]" -> "List[58]" [style=dashed, color=gray]
"ParameterDeclaration[56]" -> "Modifiers[57]" [style=dashed, color=gray]
"ParameterDeclaration[56]" -> "PrimitiveTypeAccess[59]" [style=dashed, color=gray]
"List[55]" -> "ParameterDeclaration[56]" [style=dashed, color=gray]
"Modifiers[61]" -> "List[62]" [style=dashed, color=gray]
"ParameterDeclaration[60]" -> "Modifiers[61]" [style=dashed, color=gray]
"ParameterDeclaration[60]" -> "PrimitiveTypeAccess[63]" [style=dashed, color=gray]
"List[55]" -> "ParameterDeclaration[60]" [style=dashed, color=gray]
"MethodDecl[50]" -> "List[55]" [style=dashed, color=gray]
"MethodDecl[50]" -> "List[64]" [style=dashed, color=gray]
"Block[66]" -> "List[67]" [style=dashed, color=gray]
"Opt[65]" -> "Block[66]" [style=dashed, color=gray]
"MethodDecl[50]" -> "Opt[65]" [style=dashed, color=gray]
"MethodDecl[50]" -> "Exit[66]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[50]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[28]" -> "MethodAccess[31]" [color=red, constraint=false]
"MethodAccess[31]" -> "Exit[28]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[31]" -> "GTExpr[42]" [color=red, constraint=false]
"MethodAccess[31]" -> "LTExpr[45]" [color=red, constraint=false]
"LTExpr[45]" -> "MethodAccess[31]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[45]" -> "IntegerLiteral[47]" [color=red, constraint=false]
"IntegerLiteral[47]" -> "LTExpr[45]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[47]" -> "VarAccess[46]" [color=red, constraint=false]
"VarAccess[46]" -> "IntegerLiteral[47]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[46]" -> "GTExpr[42]" [color=red, constraint=false]
"GTExpr[42]" -> "MethodAccess[31]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[42]" -> "VarAccess[46]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[42]" -> "IntegerLiteral[44]" [color=red, constraint=false]
"IntegerLiteral[44]" -> "GTExpr[42]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[44]" -> "VarAccess[43]" [color=red, constraint=false]
"VarAccess[43]" -> "IntegerLiteral[44]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[43]" -> "GTExpr[35]" [color=red, constraint=false]
"VarAccess[43]" -> "LTExpr[38]" [color=red, constraint=false]
"LTExpr[38]" -> "VarAccess[43]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[38]" -> "IntegerLiteral[40]" [color=red, constraint=false]
"IntegerLiteral[40]" -> "LTExpr[38]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[40]" -> "VarAccess[39]" [color=red, constraint=false]
"VarAccess[39]" -> "IntegerLiteral[40]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[39]" -> "GTExpr[35]" [color=red, constraint=false]
"GTExpr[35]" -> "VarAccess[39]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[35]" -> "VarAccess[43]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[35]" -> "IntegerLiteral[37]" [color=red, constraint=false]
"IntegerLiteral[37]" -> "GTExpr[35]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[37]" -> "VarAccess[36]" [color=red, constraint=false]
"VarAccess[36]" -> "IntegerLiteral[37]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[36]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VarAccess[36]" [color=blue, constraint=false, xlabel=" "] 
"Exit[66]" -> "Entry[51]" [color=red, constraint=false]
"Entry[51]" -> "Exit[66]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"MethodDecl[12]" -> "MethodDecl[50]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[26]" -> "Opt[27]" -> "Exit[28]" [style=invis]
"ParameterDeclaration[18]" -> "ParameterDeclaration[22]" [style=invis]
"Modifiers[19]" -> "PrimitiveTypeAccess[21]" [style=invis]
"Modifiers[23]" -> "PrimitiveTypeAccess[25]" [style=invis]
"AndLogicalExpr[34]" -> "OrLogicalExpr[41]" [style=invis]
"GTExpr[35]" -> "LTExpr[38]" [style=invis]
"VarAccess[36]" -> "IntegerLiteral[37]" [style=invis]
"VarAccess[39]" -> "IntegerLiteral[40]" [style=invis]
"GTExpr[42]" -> "LTExpr[45]" [style=invis]
"VarAccess[43]" -> "IntegerLiteral[44]" [style=invis]
"VarAccess[46]" -> "IntegerLiteral[47]" [style=invis]
"Entry[51]" -> "Modifiers[52]" -> "PrimitiveTypeAccess[54]" -> "List[55]" -> "List[64]" -> "Opt[65]" -> "Exit[66]" [style=invis]
"ParameterDeclaration[56]" -> "ParameterDeclaration[60]" [style=invis]
"Modifiers[57]" -> "PrimitiveTypeAccess[59]" [style=invis]
"Modifiers[61]" -> "PrimitiveTypeAccess[63]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
70", fillcolor=1]
Edges [label="Total Nr. Edges
103", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
16", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
17", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}