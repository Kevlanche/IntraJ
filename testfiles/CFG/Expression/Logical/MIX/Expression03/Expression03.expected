NodeName: Entry[13] Successors: [VarAccess[36]]
NodeName: VarAccess[36] Successors: [IntegerLiteral[37]]
NodeName: IntegerLiteral[37] Successors: [GTExpr[35]]
NodeName: GTExpr[35] Successors: [VarAccess[39], VarAccess[43]]
NodeName: VarAccess[39] Successors: [IntegerLiteral[40]]
NodeName: IntegerLiteral[40] Successors: [LTExpr[38]]
NodeName: LTExpr[38] Successors: [VarAccess[43]]
NodeName: VarAccess[43] Successors: [IntegerLiteral[44]]
NodeName: IntegerLiteral[44] Successors: [GTExpr[42]]
NodeName: GTExpr[42] Successors: [MethodAccess[31], VarAccess[46]]
NodeName: MethodAccess[31] Successors: [Exit[28]]
NodeName: Exit[28] Successors: []
NodeName: VarAccess[46] Successors: [IntegerLiteral[47]]
NodeName: IntegerLiteral[47] Successors: [LTExpr[45]]
NodeName: LTExpr[45] Successors: [MethodAccess[31]]
NodeName: Entry[51] Successors: [Exit[66]]
NodeName: Exit[66] Successors: []

NodeName: Exit[28] Predecessor: [MethodAccess[31]]
NodeName: MethodAccess[31] Predecessor: [GTExpr[42], LTExpr[45]]
NodeName: GTExpr[42] Predecessor: [IntegerLiteral[44]]
NodeName: IntegerLiteral[44] Predecessor: [VarAccess[43]]
NodeName: VarAccess[43] Predecessor: [GTExpr[35], LTExpr[38]]
NodeName: GTExpr[35] Predecessor: [IntegerLiteral[37]]
NodeName: IntegerLiteral[37] Predecessor: [VarAccess[36]]
NodeName: VarAccess[36] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: LTExpr[38] Predecessor: [IntegerLiteral[40]]
NodeName: IntegerLiteral[40] Predecessor: [VarAccess[39]]
NodeName: VarAccess[39] Predecessor: [GTExpr[35]]
NodeName: LTExpr[45] Predecessor: [IntegerLiteral[47]]
NodeName: IntegerLiteral[47] Predecessor: [VarAccess[46]]
NodeName: VarAccess[46] Predecessor: [GTExpr[42]]
NodeName: Exit[66] Predecessor: [Entry[51]]
NodeName: Entry[51] Predecessor: []
